changedetectionio/store.py

```diff
@@ -578,8 +582,28 @@ def update_5(self):
 
     # We incorrectly used common header overrides that should only apply to Requests
     # These are now handled in content_fetcher::html_requests and shouldnt be passed to Playwright/Selenium
-    def update_7(self):
+    def update_8(self):
         # These were hard-coded in early versions
         for v in ['User-Agent', 'Accept', 'Accept-Encoding', 'Accept-Language']:
             if self.data['settings']['headers'].get(v):
                 del self.data['settings']['headers'][v]
+                
+    # Generate a previous.txt for all watches that do not have one and contain history
+    def update_7(self):
```

Comment:
sorry I meant this one, should be

```suggestion
    def update_8(self):
```